Mathlib/NumberTheory/NumberField/Basic.lean

```diff
@@ -111,15 +153,11 @@ instance : IsIntegralClosure (𝓞 K) ℤ K :=
 instance [NumberField K] : IsIntegrallyClosed (𝓞 K) :=
   integralClosure.isIntegrallyClosedOfFiniteExtension ℚ
 
-theorem isIntegral_coe (x : 𝓞 K) : IsIntegral ℤ (x : K) :=
+theorem isIntegral_coe (x : 𝓞 K) : IsIntegral ℤ (algebraMap _ K x) :=
   x.2
 #align number_field.ring_of_integers.is_integral_coe NumberField.RingOfIntegers.isIntegral_coe
```

Comment:
We should probably put this before `isIntegral`, then we can simplify its proof, particularly skipping 141 `have : algebraMap _ K x = x.1 := rfl`.